onnx/defs/experiments/functions.cc

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Facebook Inc. and Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "onnx/defs/function.h"
+using namespace ONNX_NAMESPACE;
+
+static Common::Status BuildFc(std::unique_ptr<FunctionProto>* func_proto) {
```

Comment:
Let's switch MeanVarianceNormalization to be a function. FC was removed/deprecated.